airbyte-ci/connectors/metadata_service/lib/tests/test_commands.py

```diff
@@ -51,29 +51,38 @@ def mock_metadata_upload_info(
         icon_uploaded=icon_uploaded,
         icon_blob_id="icon_blob_id" if icon_uploaded else None,
         metadata_file_path=metadata_file_path,
+        doc_version_uploaded=doc_version_uploaded,
+        doc_version_blob_id="doc_version_blob_id" if doc_version_uploaded else None,
+        doc_inapp_version_uploaded=doc_inapp_version_uploaded,
+        doc_inapp_version_blob_id="doc_inapp_version_blob_id" if doc_inapp_version_uploaded else None,
+        doc_latest_uploaded=doc_latest_uploaded,
+        doc_latest_blob_id="doc_latest_blob_id" if doc_latest_uploaded else None,
+        doc_inapp_latest_uploaded=doc_inapp_latest_uploaded,
+        doc_inapp_latest_blob_id="doc_inapp_latest_blob_id" if doc_inapp_latest_uploaded else None,
     )
 
 
 # TEST UPLOAD COMMAND
 @pytest.mark.parametrize(
-    "latest_uploaded, version_uploaded, icon_uploaded",
+    "latest_uploaded, version_uploaded, icon_uploaded, doc_version_uploaded, doc_inapp_version_uploaded, doc_latest_uploaded, doc_inapp_latest_uploaded",
     [
-        (False, False, False),
-        (True, False, False),
-        (False, True, False),
-        (False, False, True),
-        (True, True, False),
-        (True, False, True),
-        (False, True, True),
-        (True, True, True),
+        (False, False, False, False, False, False, False),
+        (True, False, False, False, False, False, False),
+        (False, True, False, False, False, False, False),
+        (False, False, True, False, False, False, False),
+        (True, True, False, False, False, False, False),
+        (True, False, True, False, False, False, False),
+        (False, True, True, False, False, False, False),
+        (True, True, True, False, False, False, False),
+        (True, True, True, True, True, True, True),
     ],
 )
```

Comment:
Including every permutation of 7 different boolean parameters would be 2^7 = 128 different test cases which felt unnecessary, so I just included one test case where all are set to true

Reply:
That seems fair!

Reply:
This might be helpful in this case (I've often found myself wanting the feature in java): 
<img width="932" alt="image" src="https://github.com/airbytehq/airbyte/assets/22333512/708ca9af-d81d-467f-98e2-bc44ae22a468">
 (From https://docs.pytest.org/en/latest/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions)

Reply:
If indeed we are just enumerating all the options